<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pixelClassifyMapDialog</class>
 <widget class="QDialog" name="pixelClassifyMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../yoyirs.qrc">
    <normaloff>:/img/resources/logo.png</normaloff>:/img/resources/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SubtitleLabel" name="SubtitleLabel">
     <property name="text">
      <string>Pixel Classify Mapping</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="shpSingleRenderScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>634</width>
        <height>327</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="bgLayout">
         <item>
          <widget class="SpinBox" name="bgSpinBox">
           <property name="minimum">
            <number>-65536</number>
           </property>
           <property name="maximum">
            <number>65536</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BodyLabel" name="bgClassifyName">
           <property name="text">
            <string>Background Pixel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../yoyirs.qrc"/>
 </resources>
 <connections/>
</ui>
